### PR TITLE
community/events: Adding Ceph Days Korea event page.

### PR DESCRIPTION
**Pull request by Thingee**
Signed-off-by: Mike Perez <thingee@gmail.com>

### CODE REVIEW

**Comment by Thingee** [comment]
[:ceph-days-korea](https://github.com/Thingee/ceph.io/tree/ceph-days-korea)

**Comment by Thingee** [comment]
@zdover23 please merge if OK. I would like this page ready for Ceph Days NYC tomorrow when we announce new events.

File: src/en/community/events/2023/ceph-days-korea/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<a class="button" href="#">Registeration Coming Soon</a>
+
+## Important Dates
+
+- CFP Opens: Coming Soon!
+- CFP Closes: 2023-04-28
+- Speakers receive confirmation of acceptance: 2023-05-12
+- Schedule Announcement: 2023-05-16
+- Sponsorship Deadline: 2023-04-28
+- Event Date: 2023-06-16
```
Comment (myoungwon) [inline src/en/community/events/2023/ceph-days-korea/index.md:32]: Sorry, Event Data is 2023-06-14 (KST)

Comment (Thingee) [reply src/en/community/events/2023/ceph-days-korea/index.md]: @myoungwon Updated the date. Please review again. @zdover23, please merge after.

Comment (myoungwon) [reply src/en/community/events/2023/ceph-days-korea/index.md]: LGTM

**Review by myoungwon — APPROVED** [review]

**Pull request merged**